src/pages/candidate-details/candidate-details.html

```diff
@@ -24,39 +24,39 @@ <h4>Candidate details</h4>
     <ion-card-content>
       <ion-grid>
         <ion-row class="mes-data-row-with-separator">
-          <ion-col col-5>
+          <ion-col col-40>
```

Comment:
How come these numbers grew to 2 digits?

Reply:
The default grid size of 12 was not fine-grained enough to lay out the checkboxes correctly all on the same line. I increased the Ionic grid size to 96 and updated usages of `col-x` directive by the same scale factor to make sizings equivalent to what they were before.